src/Assignment19/AdvancedLinq/ProductsManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿// <copyright file="ProductsManager.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace AdvancedLinq
+{
+    /// <summary>
+    /// Has the methods to manage list of products.
+    /// </summary>
+    public class ProductsManager
+    {
+        /// <summary>
+        /// Gets or sets the list of products with thier details.
+        /// </summary>
+        /// <value>The products as objects</value>
+        public static List<Product> ProductsList { get; set; } = new List<Product>();
+
+        /// <summary>
+        /// Adds Default products to the list.
+        /// </summary>
+        public void IntializeWithDefaultProducts()
+        {
+            Product firstProduct = new ();
+            firstProduct.ProductId = 1;
```

Comment:
Use object initializer here or create a constructor with all properties and use it to initialize

Reply:
Object is initialized with its properties.